### DISCUSSION TITLE
Ubuntu, just updated to 1.1.7 but got 1.1.8.dev0 ???

### DISCUSSION THREAD

**Discussion by DavidJ68** [Chia-Network/chia-blockchain#6523] [Support]
Hi everyone, I'm running Ubuntu and  just updated to 1.1.7 but somehow I got 1.1.8.dev0 version. can anyone get me please?
Thanks

**Comment by rizlah** [comment]
type chia version and see what you have

**Reply by DavidJ68** [reply]
thanks for your reply, I've done that and it's 1.1.8.dev0

**Reply by MarcusB2105** [reply]
Please check issue #6497 Problem has been sovled for me.

**Comment by BrandtH22** [comment]
@DavidJ68 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
